src/config.c

```diff
@@ -713,6 +729,32 @@ void configSetCommand(client *c) {
             server.client_obuf_limits[class].soft_limit_seconds = soft_seconds;
         }
         sdsfreesplitres(v,vlen);
+    } config_set_special_field("dram-pmem-ratio") {
```

Comment:
Shoudn't there be also other directives from redis.conf: memory-alloc-policy, threshold?